index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav id="mobile-navbar">
+            <img src="assets/menu.png" alt="">
+        </nav>
+    </header>
+    <section id="headline-section">
+        <div class="container">
+            <h1 >
+                <span id="name-heading">
+                    Hey there<br>
+                I'm Abdul Salam
+                </span>
+                <span id="job-role">
+                    I',m a Software<br>
+                    Developer
+                </span>
+                
+            </h1>        
+            <p>
+                I can help you build a product, feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.
+              </p>
+           
+              <section id="social-icons">
+                <ul>
+                    <li><a href="https://twitter.com/Abdul_SalamAbro"><img src="assets/icons/twitter.png" alt=""></a></li>
+                    <li><a href="https://www.linkedin.com/in/abdul-salam-abro/"><img src="assets/icons/linkedin.png" alt=""></a></li>
+                    <li><a href="https://medium.com/abdulsalam"><img src="assets/icons/medium.png" alt=""></a></li>
+                    <li><a href="https://github.com/AbdulSalamAbro"><img src="assets/icons/github.png" alt=""></a></li>
+                    <li><a href="https://venture.angellist.com/v/Abdulsalam"><img src="assets/icons/angellist.png" alt=""></a></li>  
+                </ul>
+            </section>
```

Comment:
- [ ] (Optional) I would advise you to wrap the social media links in a `div` as the main container and not the `section` tag as you have done. As a rule, a section wraps an entire block, in this case, the introductory page. The elements inside this section should be `divs`, `articles`, `paragraphs` etc. according to what they hold